src/pages/auth/Login.jsx

```diff
@@ -1,8 +1,70 @@
+import { useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import { useAuth } from "../../Context/AuthContext";
+import { LogInHandler } from "../../authHandllers/logHandller";
+import "./auth.css";
+
 const Login = () => {
+  const { user, setUser } = useAuth();
+  const [loguser, setLoguser] = useState({ email: "", password: "" });
+  const guestUser = {
+    email: "adarshbalika@gmail.com",
+    password: "adarshBalika123",
+  };
+  const navigator = useNavigate();
   return (
-    <div>
-      Login
-      <button>Signup</button>
+    <div id="log-wrapper">
+      <div id="form-container">
+        <form
+          className="form"
+          onSubmit={(e) => {
```

Comment:
You can make submitHandler and navigate here there is no need of passing navigator in function's argument.